core/types/transaction.go

```diff
@@ -574,8 +586,11 @@ type Message struct {
 	data       []byte
 	accessList AccessList
 	isFake     bool
+
+	IsL1MessageTx bool
```

Comment:
should this be a function as defined in the `Message` interface?

Reply:
explain?